build/cite/gpkg12/citegpkg-1.2/README.rst

```diff
@@ -0,0 +1,79 @@
+Included Data
+=============
+
+This directory provides a GeoServer data directory packaged as part of our product downloads:
+
+* http://geoserver.org/display/GEOS/Download
+
+The data directory includes the following sets of information:
+
+* arc_sample
+* img_sample
+* mosaic_sample
+* nyc (giant polygon, poi, poly landmark, tiger roads)
+* sf (archsites,bugsites,restricted,roads,streams, sfdem)
+* shapefiles(states)
+* taz_shapes( cities, roads, state boundaries, water bodies)
+
+ARC Sample
+----------
+
+Data was provided by GeoSolutions with redistribution rights.
+
+IMG Sample
+----------
+
+Data was provided by GeoSolutions with redistribution rights.
+
+MOASIC Sample
+-------------
```

Comment:
MOASIC->MOSAIC

Reply:
The files are just a "git cp" (copy) of the release data directory. For WMTS tests we don't really need correctness of documentation, but it would be nice/useful for the release data dir where it came from (in a separate PR), e.g.:

https://github.com/geoserver/geoserver/blob/main/data/release/README.rst